### DISCUSSION TITLE
fullyQualifiedName returns only class name

### DISCUSSION THREAD

**Discussion by minogin** [LemonAppDev/konsist#545] [Report a Problem]
### Tell us what version of Konsist you use.

0.12.1

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

`class.fullyQualifiedName` only returns class name without package

### Actual behaviour

`class.fullyQualifiedName` only returns class name without package

### Expected behaviour

`class.fullyQualifiedName` should return fully qualified name

**Comment by minogin** [comment]
Example:

```
package com.mypackage

class S

class A(private val s1: S) {
    private val s2: S? = null
}

class KonsistTestTest : KonsistTest() {
    @Test
    fun propertyTypes() {
        val params = konsistTest.classes()
            .first { it.name == "A" }
            .properties()
            .mapNotNull { it.type }
            .map { it.fullyQualifiedName }

        println(params)
    }
}
```

Expected: `[com.mypackage.S, com.mypackage.S?]`

Actual: `[S, S?]`

Am I doing something wrong? I think it worked in some older version.

**Comment by nataliapeterwas** [comment]
Thank you for reporting the problem.

We will fix this in the next release :)

BTW if you want to check which declarations are included in your scope, you can use `print` method, e.g.
```kotlin
class KonsistTestTest : KonsistTest() {
    @Test
    fun propertyTypes() {
        konsistTest.classes()
            .first { it.name == "A" }
            .properties()
            .mapNotNull { it.type }
            .print { it.fullyQualifiedName }
    }
}
```
More information about `print` method is at the last release [v0.12.0](https://github.com/LemonAppDev/konsist/releases/tag/v0.12.0)

**Comment by igorwojda** [comment]
PR https://github.com/LemonAppDev/konsist/pull/567

BTW Docs [Debug Konsist Test](https://docs.konsist.lemonappdev.com/features/debug-konsist-test)

**Comment by igorwojda** [comment]
A new version v0.13.0 has been released. It contains the fix.
See https://github.com/LemonAppDev/konsist/releases/tag/v0.13.0
